src/rebuild.ts

```diff
@@ -151,6 +158,7 @@ export class Rebuilder implements IRebuilder {
     this.lifecycle = options.lifecycle;
     this.buildPath = options.buildPath;
     this.electronVersion = options.electronVersion;
+    this.platform = options.platform || process.platform
```

Comment:
Missing a semicolon, surprised lint didn't complain

Reply:
👍 
Double-checked all new lines for semicolons and added where I had missed them. PR updated.

Reply:
@MarshallOfSound I've opened a separate PR for fixing the semicolon lint issue 🙂 
https://github.com/electron/rebuild/pull/1154